client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -30,37 +33,46 @@ if(NOT WIN32)
 endif(NOT WIN32)
 
 # find the include files
-find_path(BOTAN_INCLUDE_DIR botan/version.h
+find_path(BOTAN_INCLUDE_DIR
+    botan/version.h
     HINTS
     ${CMAKE_FIND_ROOT_PATH}/include
-    ${BOTAN_INCLUDE_DIRS})
+    ${BOTAN_INCLUDE_DIRS}
+    ${CMAKE_INCLUDE_PATH})
 
 # locate the library
 if(WIN32)
-    set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} libbotan.lib botan.lib)
+    set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} botan)
 else(WIN32)
     if(Botan_USE_STATIC_LIBS)
-        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} libbotan.a libbotan-1.11.a)
+        set(BOTAN_LIBRARY_NAMES ${BOTAN_LIBRARY_NAMES} botan botan-1.11)
```

Comment:
Please, leave `lib` and `.a` for static libs. Otherwise, it's not different from dynamic ones